Dockerfile

```diff
@@ -9,9 +9,9 @@ RUN \
         make \
 	&& rm -rf /var/lib/apt/lists/*
 
-COPY prombench /bin/prombench
 RUN mkdir -p /prombench/components/prombench/manifests
 
+COPY prombench /prombench/prombench
 COPY Makefile /prombench/Makefile
 COPY components/prombench/nodepools.yaml /prombench/components/prombench/nodepools.yaml
 COPY components/prombench/manifests/benchmark /prombench/components/prombench/manifests/benchmark
```

Comment:
Is there no way we can mount any config files  when starting the docker container?

Reply:
In our case, k8s will mount secret `serviceaccount` at `/etc/serviceaccount/service-account.json`, when it creates the [prowjob](https://github.com/sipian/prombench/blob/refactor/components/prow/manifests/prow_internals_2.yaml#L209)

Reply:
I meant something like a volume for all config files, but lets leave it like this for now and will revisit.